File: src/main/java/org/teamvoided/transition/mappings/MappingModes.java
package org.teamvoided.transition.mappings;

import me.fzzyhmstrs.fzzy_config.util.EnumTranslatable;
import org.jetbrains.annotations.NotNull;
import org.teamvoided.transition.Transition;

public enum MappingModes implements EnumTranslatable {
    OFF,
    ON_LOAD,
    CONTINUOUS;

    @NotNull
    @Override
    public String prefix() {
        return Transition.MODID + ".main.mode";
    }
}

File: src/main/kotlin/org/teamvoided/transition/config/MappingModes.kt
package org.teamvoided.transition.config

import me.fzzyhmstrs.fzzy_config.util.EnumTranslatable
import org.teamvoided.transition.Transition.MODID

enum class MappingModes : EnumTranslatable {
    OFF,
    ON_LOAD,
    CONTINUOUS;

    override fun prefix(): String = "$MODID.main.mode"
}
